m0t1fs/linux_kernel/st/m0t1fs_common_inc.sh

```diff
@@ -866,6 +866,7 @@ function run()
 {
 	echo "# $*"
 	eval $*
+	return $?
```

Comment:
@sanjognaik  we need this return statement? Please check once. Thanks

Reply:
No. It's not required. Removed it. Thanks.